config/env/development.js

```diff
@@ -64,9 +64,12 @@ module.exports = {
     autoLogoff: false,
     defaultGroup: '',
     awsCache: 0,
+    sessionDuration: 600,
+    creditLimit: '0.001',
```

Comment:
The default value of a configuration variable should serve most use case. In this case, I think credit limit should be disabled by default.